templates/security/cc.html

```diff
@@ -0,0 +1,114 @@
+  </div>
+  <div class="row">
+    <div class="col-8">
+      <h2>Where is Common Criteria accepted?</h2>
+      <p>Internationally a Common Criteria certification is accepted by members of the <a href="https://www.commoncriteriaportal.org/ccra/members/" class="p-link--external">CCRA</a> agreement and the EU <a href="https://www.sogis.eu/" class="p-link--external">SOGIS</a> members.</p>
+    </div>
+  </div>
+  <div class="u-fixed-width">
+    <hr class="p-separator" />
+  </div>
+  <div class="row">
+    <div class="col-8">
+      <h2>What gets certified in Ubuntu under Common Criteria?</h2>
+      <p>Ubuntu 18.04 LTS and 16.04 LTS have both been evaluated to assurance level EAL2 through <a href="http://www.fmv.se/en/Our-activities/CSEC---The-Swedish-Certification-Body-for-IT-Security/" class="p-link--external">CSEC – The Swedish Certification Body for IT Security</a>. The evaluation testing was performed by atsec Information Security.  The following table provides a summary of the releases and platforms that have been certified.</p>
+    </div>
+    <div class="col-4">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/961a1ad1-csec-logo-removebg-preview.png",
+        alt="",
+        width="180",
+        height="205",
+        hi_def=True,
+        loading="auto|lazy"
+        ) | safe
+      }}
+    </div>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="u-fixed-width">
+    <table>
+      <thead>
+        <tr>
+          <th>Ubuntu version</th>
+          <th>Platform</th>
+          <th>Certification report</th>
+          <th>Additional information</th>
+        </tr>
+      </thead>
+      <t-body>
+        <tr>
+          <td>Ubuntu 16.04 LTS</td>
+          <td>x86_64, IBM Power8 and IBM Z</td>
+          <td><a class="p-link--external" href="https://www.commoncriteriaportal.org/files/epfiles/Certification%20Report%20Ubuntu%20LTS%2016.04.4.pdf">16.04.4</a></td>
+          <td><a class="p-link--external" href="https://security-certs.docs.ubuntu.com/en/cc">Installation instructions</a></td>
+        </tr>
+      </t-body>
+      <t-body>
+        <tr>
+          <td>Ubuntu 18.04 LTS</td>
+          <td>x86_64 and IBM Z</td>
+          <td><a class="p-link--external" href="https://www.commoncriteriaportal.org/files/epfiles/Certification%20Report%20-%20Canonical%20Ubuntu%20Server%2018.04%20LTS.pdf">18.04.4</a></td>
+          <td><a class="p-link--external" href="https://security-certs.docs.ubuntu.com/en/cc">Installation instructions</a></td>
+        </tr>
+      </t-body>
+    </table>
+    <p>
+      <a class="p-button--positive js-invoke-modal" href="/support/get-in-touch">Contact us</a>
+    </p>
+  </div>
+</section>
+
+<!-- Set default Marketo information for contact form below-->
+<div class="u-hide" id="contact-form-container" data-form-location="/shared/forms/interactive/esm" data-form-id="3764" data-lp-id="" data-return-url="https://www.ubuntu.com/security/thank-you?product=esm" data-lp-url="">
```

Comment:
can you copy this from here please?
https://github.com/canonical-web-and-design/ubuntu.com/pull/9862/files

(here you're using the esm page but it's the support one we need. Make sure the data-form-id is the same as the other page too)

Reply:
@bethcollins92 There we go :+1: